style.css

```diff
@@ -322,3 +322,87 @@ a {
   font-size: 32px;
   color: #fff;
 }
+
+/* FORM */
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+form p {
+  font-family: "Inter", serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+  height: 216px;
+  width: 84%;
+  margin: 0 24px;
+  padding-top: 120px;
+  margin-bottom: 73px;
+}
+
+input[type="text"],
+input[type="email"] {
+  font-family: "Inter", serif;
+  margin-top: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  width: 295px;
+  border: #979493;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+textarea {
+  margin: 24px;
+  width: 327px;
+  height: 114px;
+  border: #d4cfc9;
+  box-sizing: border-box;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.get {
+  font-family: "Inter", sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  background: #f55800;
+  color: #fff;
+  margin: 24px 122px;
+  align-items: center;
+  padding: 12px;
+  width: 131px;
+  height: 48px;
+}
```

Comment:
- [ ] Please add `hover` and `pressed` button interaction styles by adding `hover` and `active` pseudo classes with the given Figma design.

![image](https://user-images.githubusercontent.com/68796232/186276122-138482e0-4965-4d41-805f-d6dbaf3dc12f.png)